css/giving-rides.css

```diff
@@ -0,0 +1,304 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+body,
+h1,
+h2,
+h3,
+h4,
+p,
+ul[class],
+ol[class],
+li,
+figure,
+figcaption,
+blockquote,
+dl,
+dd {
+  margin: 0;
+}
+
+html{
+    /* font-size:62.5%; */
+    text-decoration: none;
+    font-family: 'Open Sans', sans-serif;
+}
+ul{
+    list-style-type: none;
+    margin: 0;
+}
+
+.important-info{
+    width: auto;
+    height: 8.5rem;
+    background-color: #DFF4F3;
+    font-size: 2.6rem;
+    display: flex;
+    align-items: center;
+    padding-left:11rem; 
+}
+#info-list li::before{
+    content: "\25A0";
+    color: #5C6E93;
+    display: inline-block; /* Needed to add space between the bullet and the text */
+    width: 5rem; /* Also needed for space (tweak if needed) */
+    padding-top: 2rem;
+    margin-left: -5rem; /* Also needed for space (tweak if needed) */
+}
+#info-list{
```

Comment:
If you do ul.info-list and make sure to delete all the default styles you had above, that should allow you to use a class. ul.info-list is a more specific selector that can override the default